extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -377,21 +377,12 @@ class MembershipsButtonEdit extends Component {
 				</PanelBody>
 			</InspectorControls>
 		);
-		const blockClasses = classnames( className, [
-			'wp-block-button__link',
-			'components-button',
-			'is-primary',
-			'is-button',
-			`align${ align }`,
-		] );
-		const blockContent = (
-			<SubmitButton
-				className={ blockClasses }
-				submitButtonText={ this.props.attributes.submitButtonText }
-				attributes={ this.props.attributes }
-				setAttributes={ this.props.setAttributes }
-			/>
-		);
+		const additionalClasses = [ 'components-button', 'is-primary', 'is-button', `align${ align }` ];
+		const buttonClasses = ! isEmpty( this.props.attributes.submitButtonClasses )
+			? this.props.attributes.submitButtonClasses.split( ' ' ).concat( additionalClasses )
+			: [ 'wp-block-button__link', ...additionalClasses ];
```

Comment:
I wonder if it's worth extracting this to a function to just make it a bit easier to read.

Reply:
I pushed a commit to do this...